invenio_checks/ext.py

```diff
@@ -31,3 +36,19 @@ def init_config(self, app):
         for k in dir(config):
             if k.startswith("CHECKS_"):
                 app.config.setdefault(k, getattr(config, k))
+
+    def service_configs(self, app):
+        """Customized service configs."""
+
+        class ServiceConfigs:
+            checks_configs = ChecksConfigServiceConfig.build(app)
+
+        return ServiceConfigs
+
+    def init_services(self, app):
+        """Initialize the service and resource for Requests."""
+        service_configs = self.service_configs(app)
+
+        self.checks_configs_service = CheckConfigService(
+            config=service_configs.checks_configs,
+        )
```

Comment:
nit: we can inline and shorten these calls and naming since unlike e.g. the RDM records services, the Checks services are going to be much simpler (see also [`invenio-jobs`](https://github.com/inveniosoftware/invenio-jobs/blob/070d0de311a39d1e3bd4a3ee6c060afb05c5d692/invenio_jobs/ext.py#L60)):

```python
self.service = CheckConfigService(
    config=ChecksConfigServiceConfig.build(app),
)
```